integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java

```diff
@@ -365,8 +365,7 @@ void testCreateDeleteCreateOperatorButNotDomain() {
 
       //check if the operator can still manage domain1
       int replicaCountDomain1 = 2;
-      assertTrue(scaleClusterWithRestApi(domain1Uid, clusterName,replicaCountDomain1 - 1,
-          externalRestHttpsPort,opNamespace, opServiceAccount),
+      assertTrue(scaleCluster(domain1Uid + "-" + clusterName, domain1Namespace, replicaCountDomain1 - 1),
```

Comment:
Since the REST api is not used to scale cluster this code can be removed.
`      createRouteForOKD("external-weblogic-operator-svc", opNamespace);
      setTlsTerminationForRoute("external-weblogic-operator-svc", opNamespace);
      int externalRestHttpsPort = getServiceNodePort(opNamespace, "external-weblogic-operator-svc");
      assertNotEquals(-1, externalRestHttpsPort,
          "Could not get the Operator external service node port");
      logger.info("externalRestHttpsPort {0}", externalRestHttpsPort);`